source/feature/openvswitch/native-openvswitch.html.md

```diff
@@ -25,6 +25,7 @@ OVS is to be fully integrated (natively) into oVirt through VDSM, becoming the d
 ## Overview
 The introduction of OVS as an additional network switching solution raises the need to generalize and abstract the requirements from the system networking services and define a spec which considers both options (OVS & native Linux networking).
 The OVS model of arranging networking entities is different from the native Linux networking, but is closer to industry networking modeling.
+Direct connection of VM to the network cannot be done by OvS but it is supported only with OVN.
```

Comment:
This is too succinct. Please try something along the following lines - after all, autodefine is *your* feature!

Direct connection of VM to the network cannot be done by OvS. In order to connect a VM to an OvS bridge, a user must first define an OvS VM network. This automatically defines an OVN network that. The VM can be attached to the OVN network which is based on the OvS one.

Reply:
Done. I have hopefully described some of the missing features in more user friendly way.